libraries/rush-lib/src/api/RushConfiguration.ts

```diff
@@ -340,12 +358,13 @@ export class RushConfiguration {
   public readonly subspacesFeatureEnabled: boolean;
 
   /**
-   * If true, the `variants` field is present in rush.json.
+   * The filename of the variant dependency data file.  By default this is
+   * called 'current-variant.json' resides in the Rush common folder.
+   * Its data structure is defined by ICurrentVariantJson.
    *
-   * @internal
-   * @deprecated - Remove when the field is removed from the rush.json schema.
+   * Example: `C:\MyRepo\common\temp\current-variant.json`
    */
-  public readonly _hasVariantsField: boolean;
+  public readonly currentVariantJsonFilePath: string;
```

Comment:
What if we aren't using the variants feature? Would this still be a string?

Reply:
To make sure turning the feature on and off is handled correctly, I think we should always write the file. That's what used to happen.